packages/battery/battery/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.11
+
+* Update the example app: remove the depreciated `RaisedButton` and `FlatButton` widgets.
```

Comment:
Isn't it "deprecated", instead of "depreciated"?

Reply:
fixed